src/scripts/userview/userview_modal.js

```diff
@@ -95,7 +95,7 @@ export function oilShowPreferenceCenter() {
           let consentData = getSoiConsentData();
           let currentPrivacySettings;
           if (consentData) {
-            currentPrivacySettings = consentData.getPurposesAllowed();
+            currentPrivacySettings = [...consentData.getPurposesAllowed(), ...getSoiCustomConsents()];
```

Comment:
Just making sure, this will be transpiled, right? IE support and all that stuff

Reply:
Good shout. There's a transpiler in there. oil.js is advertised as IE9 friendly, so the `let`'s would have to go as well.